src/content/developers/tutorials/how-to-write-and-deploy-an-nft.md

```diff
@@ -0,0 +1,334 @@
+
+A ContractFactory in ethers.js is an abstraction used to deploy new smart contracts, so MyNFT here is a factory for instances of our NFT contract. When using the hardhat-ethers plugin ContractFactory and Contract instances are connected to the first signer by default.
+
+    const myNFT = await MyNFT.deploy();
+
+Calling deploy() on a ContractFactory will start the deployment, and return a Promise that resolves to a Contract. This is the object that has a method for each of our smart contract functions.
+
+## Step 16: Deploy our contract
+
+We’re finally ready to deploy our smart contract! Navigate back to the root of your project directory, and in the command line run:
+
+    npx hardhat run scripts/deploy.js --network ropsten
+
+You should then see something like:
+
+    Contract deployed to address: 0x81c587EB0fE773404c42c1d2666b5f557C470eED
+
+If we go to the [Ropsten etherscan](https://ropsten.etherscan.io/) and search for our contract address we should able to see that it has been deployed successfully. The transaction will look something like this:
+
+![View your transaction address on Etherscan](https://cdn-images-1.medium.com/max/5200/0*qlORqkJ0KBb39Y1m.png)
+
+The From address should match your Metamask account address and the To address will say “Contract Creation.” If we click into the transaction, we’ll see our contract address in the To field:
+
+![View your contract address on Etherscan](https://cdn-images-1.medium.com/max/5200/0*yu-kGK524F12jFvZ.png)
+
+Yasssss! You just deployed your NFT smart contract to the Ethereum chain 🎉
+
+To understand what’s going on under the hood, let’s navigate to the Explorer tab in our [Alchemy dashboard ](https://dashboard.alchemyapi.io/explorer). If you have multiple Alchemy apps make sure to filter by app and select “MyNFT”.
+
+![View calls made “under the hood” with Alchemy’s Explorer Dashboard](https://cdn-images-1.medium.com/max/5200/0*9vAulgWSnJJ_Xf4u.png)
+
+Here you’ll see a handful of JSON-RPC calls that Hardhat/Ethers made under the hood for us when we called the .deploy() function. Two important ones to call out here are [eth_sendRawTransaction](https://docs.alchemyapi.io/alchemy/documentation/alchemy-api-reference/json-rpc#eth_sendrawtransaction), which is the request to actually write our smart contract onto the Ropsten chain, and [eth_getTransactionByHash](https://docs.alchemyapi.io/alchemy/documentation/alchemy-api-reference/json-rpc#eth_gettransactionbyhash) which is a request to read information about our transaction given the hash (a typical pattern when sending transactions). To learn more about sending transactions, check out this tutorial on [sending transactions using Web3](https://docs.alchemyapi.io/alchemy/tutorials/sending-transactions-using-web3-and-alchemy).
+
+That’s all for Part 1 of this tutorial. In Part 2, we’ll actually interact with our smart contract by minting an NFT, and in Part 3 we’ll show you how to view your NFT in your Ethereum wallet!
```

Comment:
Would add a link here to go to the next part

Don't worry about it being in a separate PR, they will end up merged together. Make sure to update the files in the other two PRs to `part-2-name/index.md` instead of `part-2-name.md` (for example), and in this PR, add a link to the bottom to `/developers/tutorials/part-2-name/`